components/recap/mentionFancy.js

```diff
@@ -30,7 +30,7 @@ const Mention = memo(({ username }) => {
   }, [])
   return (
     <a sx={{textDecoration:'none'}} href={`https://scrapbook.hackclub.com/${username}`}>
-      <a sx={{textDecoration:'none'}} className="mention post-text-mentionfancy">
+      <a className="mention post-text-mentionfancy" href={`https://scrapbook.hackclub.com/${username}`}>
```

Comment:
`<a>` tags should never be nested—I think you can remove the higher-level one entirely. & fyi, `sx` will be ignored here since you’re not using Theme UI’s jsx pragma—you’ll need to use their `Link` component (you can do `import { Link as StyledLink }` or something) to use Theme UI styling

Reply:
ngl, I have no clue what this piece of code does.....

Reply:
I was just too scared to delete it